clients/admin-ui/src/features/common/nav/v2/nav-config.tsx

```diff
@@ -19,12 +21,13 @@ export type NavConfigRoute = {
 
 export type NavConfigGroup = {
   title: string;
+  icon: React.ReactNode;
```

Comment:
I think we typically prefer importing react types
```
import { ReactNode } from `react`;
```
```suggestion
  icon: ReactNode;
```

Reply:
updated